cmd/pool/account.go

```diff
@@ -417,6 +418,17 @@ var depositAccountCommand = cli.Command{
 			Usage: "the fee rate expressed in sat/vbyte that " +
 				"should be used for the deposit",
 		},
+		cli.Uint64Flag{
+			Name: accountExpiryAbsolute,
+			Usage: "the new block height at which this account " +
```

Comment:
nit: remove first occurrence of "at"?

Reply:
Could also define the whole flag as a variable and then use it twice?

Reply:
done (it was shared with other cmds too)